app/models/planning_application_policy_class.rb

```diff
@@ -12,13 +12,14 @@ class PlanningApplicationPolicyClass < ApplicationRecord
     validates :reporting_types, presence: true
   end
 
+  after_create :initialize_planning_application_policy_sections
+
   delegate :section, :name, :url, :description, to: :policy_class
-  delegate :planning_application_policy_sections, to: :planning_application
```

Comment:
Can't delegate this because the policy sections will be for all policy classes on this application, not just this class.